bin/check-azurerm-monitor-metric.rb

```diff
@@ -127,12 +127,16 @@ class CheckAzurermMonitorMetric < Sensu::Plugin::Check::CLI
          short: '-f FILTER',
          long: '--filter FILTER'
 
-  option :aggregation,
-         description:  'Aggregation.  This can be Average, Count, Maximum, Minimum, Total',
+  option :request_aggregation,
+         description:  'Used as a parameter to the HTTP request sent to Azure.   This can be Average, Count, Maximum, Minimum, Total',
          short: '-a aggregation',
          long: '--aggregation aggregation',
          default: 'average'
 
+  option :aggregate_results,
+         description: 'Aggregate the result data points to compare against alert conditions.   This can be Average, Count, Maximum, Minimum, Total',
+         long: '--aggregate_results aggregation_type'
+
```

Comment:
we probably should default to something does `average` sound like a good default?

also you can use: `in: %(average count maximum minimum count total )` and can be applied to both of them. This should help avoid someone using a non valid value.

Reply:
In the behavior of the code I wrote, if no `aggregate_results` option is set, then the "old" code will be executed to enable backward compatibility.

I can/will add the `in`

Reply:
`in: %(average count maximum minimum count total)`
Would you be able to give me the exact syntax?
does not seem to work. 

`  option :aggregate_results,
         description: 'Aggregate the result data points to compare against alert conditions.   This can be average, count, maximum, minimum, total',
         long: '--aggregate_results aggregation_type'
         in: %(average count maximum minimum count total)`

Reply:
sorry I meant `%w()`, I will edit the previous suggestion.

Reply:
this is fine as it is, I must have missed the conditional for `"none"`